tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/icon_keypoints_block.html

```diff
@@ -1,22 +1,24 @@
 {% load wagtailcore_tags wagtailimages_tags %}
-<div>
-    {# Section heading #}
-    <h2>{{ value.title }}</h2>
+<div class="grid__keypoints keypoints">
+    {% if value.title %}
+        <h2 class="heading heading--two">{{ value.title }}</h2>
+    {% endif %}
+    {% if value.intro %}
+        <div class="text text--five">{{ value.intro|richtext }}</div>
+    {% endif %}
 
-    {# Section intro #}
-    <div>{{ value.intro|richtext }}</div>
-
-    {# Key points #}
-    <ul>
+    <ul class="keypoints__list">
         {% for key_point in value.key_points %}
-            <li>
-                {% include "patterns/atoms/icons/icon.html" with name=key_point.icon classname=key_point.icon %}
-                <p>
-                    <b>Icon:</b> {{ key_point.icon }}<br>
-                    <b>Icon Label:</b> {{ key_point.icon_label }}<br>
-                    <b>Heading:</b> {{ key_point.heading }}
-                </p>
-                <div>{{ key_point.description|richtext }}</div>
+            <li class="keypoints__item">
+                <div class="keypoints__wrapper">
+                    <p class="keypoints__label">{{ key_point.icon_label }}</p>
+                    {% include "patterns/atoms/icons/icon.html" with name=key_point.icon classname="keypoints__icon key_point.icon" %}
```

Comment:
Question: Do we need to pass `key_point.icon` as a classname here?

Reply:
good catch - definitely don't need that